app/services/profile_edit_service.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class ProfileEditService < ApplicationService
```

Comment:
let's have profiles' services under `profiles` module. Then it will be easier to find something when your project is growing.
```
module Profiles
  class Edit < ApplicationService
  ...
  end
end
```